mage_ai/api/resources/PipelineResource.py

```diff
@@ -79,7 +79,10 @@ def query_pipeline_schedules(pipeline_uuids):
                     a.status,
                     a.updated_at,
                 ]).
-                filter(a.pipeline_uuid.in_(pipeline_uuids))
+                filter(
+                    a.pipeline_uuid.in_(pipeline_uuids),
+                    a.repo_path == get_repo_path(),
```

Comment:
should we use project_uuid here as well? will there be duplicate repo_path in subprojects?

Reply:
i already made the change to save the repo path in previous PR, so if i change it to project_uuid here, it may affect existing projects.

the repo_paths for the subprojects should be different because each subproject should have a different project name.